wadas/domain/configuration.py

```diff
@@ -0,0 +1,138 @@
+"""WADAS configuration module."""
+
+import logging
+import os
+
+import keyring
+import yaml
+
+from wadas.domain.actuator import Actuator
+from wadas.domain.ai_model import AiModel
+from wadas.domain.camera import Camera, cameras
+from wadas.domain.email_notifier import EmailNotifier
+from wadas.domain.fastapi_actuator_server import FastAPIActuatorServer
+from wadas.domain.feeder_actuator import FeederActuator
+from wadas.domain.ftp_camera import FTPCamera
+from wadas.domain.ftps_server import FTPsServer
+from wadas.domain.notifier import Notifier
+from wadas.domain.operation_mode import OperationMode
+from wadas.domain.roadsign_actuator import RoadSignActuator
+from wadas.domain.usb_camera import USBCamera
+
+logger = logging.getLogger(__name__)
+WADAS_VERSION = 0.1
```

Comment:
TODO: use a better way to track system version. In particular we should have a unique point in the packet for this. The standard way is to have a `__version__` variable in the module `__init__.py` file and then you get it by using `wadas.__version__`